app/src/newdocs/content/marketplace.mdx

```diff
@@ -1,11 +1,15 @@
 import ScrollableAnchor from 'react-scrollable-anchor'
+import { Link } from 'react-router-dom'
 
+import links from '$shared/../links'
 import docsStyles from '$newdocs/components/DocsLayout/docsLayout.pcss'
 
 <ScrollableAnchor id="introduction-marketplace"><div>
 
 ## Introduction to the Marketplace
---content--
+The [marketplace](https://streamr.com/marketplace) is a Streamr developed application that handles the buying and selling of real-time data on the Streamr platform. The marketplace is filled with both free products and products created by data producers that wish to monetise their data. The currency of the marketplace is the <Link to={links.newdocs.dataToken}>DATA token</Link>.
```

Comment:
Question: How can we keep links DRY in the docs? Why not put all (including external) links in `$shared/../links` instead of hard-coding them inline?

Reply:
The links are DRY in `$shared/../links` and include external links, so I unnecessarily hard linked the external link here. Will fix shortly.